src/scripts.js

```diff
@@ -89,6 +129,68 @@ function displayHydrationData() {
 });
 }
 
+function displaySleepData() {
+  let daySleep = sleepUser1.showHoursSleptByDate(returnUserSelectedDate());
+  let qualitySleepDay = sleepUser1.showSleepQualityByDate(returnUserSelectedDate())
+  hoursSleptDay.innerText = `Hours slept on this date: ${daySleep}`;
+  sleepQualityDay.innerText = `Quality hours slept on this date: ${qualitySleepDay}`;
+  let userAvgSleepHours = sleepUser1.calcAllTimeSleepAvg('hoursSlept');
+  let userAvgQualityHours = sleepUser1.calcAllTimeSleepAvg('sleepQuality');
+  avgSleepHours.innerText = `Total average sleep hours: ${userAvgSleepHours}`;
+  avgSleepQuality.innerText = `Total average sleep quality: ${userAvgQualityHours}`;
+  const weeklySleepDays = Object.keys(sleepUser1.showHoursSleptByWeek(returnUserSelectedDate()));
+  const weeklySleepHours = Object.values(sleepUser1.showHoursSleptByWeek(returnUserSelectedDate()));
+  const weeklyQualitySleepHours = Object.values(sleepUser1.showSleepQualityByWeek(returnUserSelectedDate()));
+  let userSleepChart = new Chart(sleepChart, {
+    type: 'bar',
+    data: {
```

Comment:
refactoring idea: separate chart function with parameters to take in different data sets?